model/project_aliases.go

```diff
@@ -484,7 +484,6 @@ func UpsertAliasesForProject(aliases []ProjectAlias, projectId string) error {
 		catcher.Add(aliases[i].Upsert())
 	}
 	grip.Debug(message.WrapError(catcher.Resolve(), message.Fields{
```

Comment:
would 
`grip.DebigWhen(catcher.hasErrors, ...) `
make it cleaner or does it already not print when the catcher has nothing?

Reply:
Already doesn't print when the catcher has nothing 👍